### DISCUSSION TITLE
EL2 My Open Math

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/precalculus#387] [General]
Reviewer: @tdegeorge 

Questions 5-11 are good basic graphing ones, but they don't have them plot the asymptote. Questions 12-19 do have them graph the asymptote, and half of them also have them state the equation of the asymptote. I'm leaning toward removing 5-11 to be consistent about emphasizing the asymptote. Thoughts?

I couldn't find any where they actually graph any with base e, but I'm assuming that's because the graphing method has students place integer points. I did see some where students can upload a file with their graph but I'm not a fan of the ones where the feedback isn't immediate for homework. I did add some where they find the range and asymptote for ones with e.

**Comment by tdegeorge** [comment]
I agree - I would prefer for students to be able to think about and/or graph the asymptote when doing these graphs. I would eliminate 5-11.

**Comment by tdegeorge** [comment]
I tried to search a couple different things and I couldn't find any problems with graphing with base e either....

I think what you have at the end is fine though since they have to find characteristics with base e.
